docs/en/ingest-management/getting-started.asciidoc

```diff
@@ -192,6 +192,78 @@ to the data stream.
 
 //TODO: Add dashboard.
 
+[discrete]
+== Data from Splunk (Experimental)
+
+Apache, AWS Cloudtrail, Nginx, and Zeek integrations offer the ability
+to pull logs from a Splunk Enterprise instance.  This sections covers
+configuration & limitations of this option.
+
+To ingest Nginx data from Splunk the following steps are performed.
+The options are the same for Apache, AWS Cloudtrail and Zeek.
+
+. Find the Nginx integration and begin adding it as described in <<<add-nginx-integration>>>.
+
+. Enable "Collect logs from third-party REST API" and disable both "Collect
+logs from Nginx instances" and "Collect metrics from Nginx instances".
++
+[role="screenshot"]
+image::images/kibana-fleet-third-party-rest-api.png[{fleet} showing enabling third-party REST API]
+
+. Enter the required information to connect to the Splunk Enterprise REST API.
++
+The URL of the Splunk Enterprise Server must include the scheme (http or https),
+the IP address or hostname of the Splunk Enterprise Server, and the the port the
+REST API is listening on.
++
+The Splunk username and password must be of a user with a role or
+capability to use REST API endpoints.  Administrative users have these
+permissions by default.
++
+SSL Configuration is available under the "Advanced options".  These may be necessary
+if Splunk Enterprise server uses self signed certificates.  See
+https://www.elastic.co/guide/en/beats/filebeat/current/configuration-ssl.html[SSL Options]
+for valid configuration options.
++
+[role="screenshot"]
+image::images/kibana-fleet-third-party-rest-settings.png[{fleet} showing enabling third-party REST API settings]
+
+. For each type of log file enter the interval and Splunk search string.
++
+The interval is expressed as a
```

Comment:
Maybe worth adding some additional guidance here. If you are ingesting a large (subjective, I know) volume of data via the Splunk REST API, we recommend increasing the interval to avoid performance degradation....